be/src/exec/aggregator.cpp

```diff
@@ -809,6 +808,51 @@ Status Aggregator::output_chunk_by_streaming(Chunk* input_chunk, ChunkPtr* chunk
     return Status::OK();
 }
 
+Status Aggregator::convert_to_spill_format(Chunk* input_chunk, ChunkPtr* chunk) {
```

Comment:
I think this interface is not necessary, I have modified convert_hash_map_to_chunk interface to support convert hash map to intermediate format in my later change

Reply:
This function is introduced because the input may be serialize format and the output may be serialize format during the 3/4 stage aggregation. It is hard to get the correct result by output_chunk_by_streaming